pkg/api/v20231122/openshiftcluster_example.go

```diff
@@ -0,0 +1,90 @@
+package v20231122
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"github.com/Azure/ARO-RP/pkg/api"
+)
+
+func exampleOpenShiftCluster() *OpenShiftCluster {
+	doc := api.ExampleOpenShiftClusterDocument()
+	doc.OpenShiftCluster.Properties.WorkerProfilesStatus = nil
+	return (&openShiftClusterConverter{}).ToExternal(doc.OpenShiftCluster).(*OpenShiftCluster)
+}
+
+// ExampleOpenShiftClusterPatchParameter returns an example OpenShiftCluster
+// object that an end-user might send to create a cluster in a PATCH request
+func ExampleOpenShiftClusterPatchParameter() interface{} {
+	oc := ExampleOpenShiftClusterPutParameter().(*OpenShiftCluster)
+	oc.Location = ""
+	oc.SystemData = nil
+	oc.Properties.WorkerProfilesStatus = nil
+	return oc
+}
+
+// ExampleOpenShiftClusterPutParameter returns an example OpenShiftCluster
+// object that an end-user might send to create a cluster in a PUT request
+func ExampleOpenShiftClusterPutParameter() interface{} {
+	oc := exampleOpenShiftCluster()
+	oc.ID = ""
+	oc.Name = ""
+	oc.Type = ""
+	oc.Properties.ProvisioningState = ""
+	oc.Properties.ClusterProfile.Version = ""
+	oc.Properties.ClusterProfile.FipsValidatedModules = FipsValidatedModulesEnabled
+	oc.Properties.ConsoleProfile.URL = ""
+	oc.Properties.APIServerProfile.URL = ""
+	oc.Properties.APIServerProfile.IP = ""
+	oc.Properties.IngressProfiles[0].IP = ""
+	oc.Properties.MasterProfile.EncryptionAtHost = EncryptionAtHostEnabled
+	oc.Properties.NetworkProfile.LoadBalancerProfile = &LoadBalancerProfile{
+		ManagedOutboundIPs: &ManagedOutboundIPs{
+			Count: 1,
+		},
+	}
+	oc.SystemData = nil
+
+	return oc
+}
+
+// ExampleOpenShiftClusterResponse returns an example OpenShiftCluster object
+// that the RP might return to an end-user in a GET response
+func ExampleOpenShiftClusterGetResponse() interface{} {
+	oc := exampleOpenShiftCluster()
+	oc.Properties.ClusterProfile.PullSecret = ""
+	oc.Properties.ServicePrincipalProfile.ClientSecret = ""
+	oc.Properties.NetworkProfile.LoadBalancerProfile = &LoadBalancerProfile{
```

Comment:
diff: missing `oc.Properties.WorkerProfilesStatus = nil` that's in stable.

Reply:
Added, thanks.